sentry-core/src/hub.rs

```diff
@@ -0,0 +1,163 @@
+use crate::{Client, Event, IntoBreadcrumbs, Level, Scope, ScopeGuard, Uuid};
+
+/// The central object that can manages scopes and clients.
+///
+/// This can be used to capture events and manage the scope. This object is
+/// internally synchronized so it can be used from multiple threads if needed.
+/// The default hub that is available automatically is thread local.
+///
+/// See the
+/// [Unified API](https://docs.sentry.io/development/sdk-dev/unified-api/#hub)
+/// documentation for further details.
+///
+/// In most situations developers do not need to interface with the Hub
+/// directly. Instead, toplevel convenience functions are exposed that will
```

Comment:
> In most situations developers do not need to interface with the Hub directly.

I've come to learn that this is not true. Especially with async/await or concurrent software, this is even the common case.